File: src/constants/events.ts
/** Tauri 事件名常量（与 Rust 端 src-tauri/src/events.rs 保持一致） */

// === 网络状态 ===
export const NETWORK_STATUS_CHANGED = "network-status-changed";
export const DEVICES_CHANGED = "devices-changed";

// === 配对 ===
export const PAIRING_REQUEST_RECEIVED = "pairing-request-received";
export const PAIRED_DEVICE_ADDED = "paired-device-added";

// === 传输 ===
export const TRANSFER_OFFER = "transfer-offer";
export const TRANSFER_PROGRESS = "transfer-progress";
export const TRANSFER_COMPLETE = "transfer-complete";
export const TRANSFER_FAILED = "transfer-failed";

File: src-tauri/src/events.rs
//! Tauri 事件名常量
//!
//! 所有后端 → 前端的事件名集中定义，避免硬编码字符串散落各模块。

// === 网络状态 ===
pub const NETWORK_STATUS_CHANGED: &str = "network-status-changed";
pub const DEVICES_CHANGED: &str = "devices-changed";

// === 配对 ===
pub const PAIRING_REQUEST_RECEIVED: &str = "pairing-request-received";
pub const PAIRED_DEVICE_ADDED: &str = "paired-device-added";

// === 传输 ===
pub const TRANSFER_OFFER: &str = "transfer-offer";
pub const TRANSFER_PROGRESS: &str = "transfer-progress";
pub const TRANSFER_COMPLETE: &str = "transfer-complete";
pub const TRANSFER_FAILED: &str = "transfer-failed";
